docs/anime.md

```diff
@@ -253,6 +253,18 @@ pageClass: routes
 
 </Route>
 
+## iwara
+
+### 用户
+
+&lt;Route author="Fatpandac" example="/iwara/users/kelpie/video" path="/iwara/users/:username/:type?" :paramsDesc="['', 'type 默认为 video'] radar="1" rssbud="1">
```

Comment:
```suggestion
<Route author="Fatpandac" example="/iwara/users/kelpie/video" path="/iwara/users/:username/:type?" :paramsDesc="['', 'type 默认为 video'] radar="1" rssbud="1">
```

Reply:
Please provide description to `username`

Reply:
Thx. I'll add a description for `username` late.